pg_search/src/telemetry/posthog.rs

```diff
@@ -0,0 +1,70 @@
+use pgrx::*;
+use serde_json::json;
+use std::env;
+use std::fs;
+
+pub unsafe fn init() {
+    // Read whether telemetry was already handled at the ParadeDB level
+    let telemetry_handled = match fs::read_to_string("/tmp/telemetry") {
+        Ok(content) => content.trim().to_string(),
+        Err(_) => String::from("false"),
+    };
+
+    // If telemetry was not already handled at the ParadeDB level, this is a standalone deployment
+    // of pg_search, and we should send telemetry
+    if telemetry_handled != "true" {
+        // We only send telemetry in the standalone deployment if TELEMETRY is set to true
+        let telemetry = env::var("TELEMETRY").unwrap_or_else(|_| String::from("false"));
+        if telemetry == "true" {
+            if let Ok(api_key) = env::var("POSTHOG_API_KEY") {
+                if let Ok(posthog_host) = env::var("POSTHOG_HOST") {
+                    if let Ok(commit_sha) = env::var("COMMIT_SHA") {
+                        // The endpoint for sending events to PostHog
+                        let endpoint: String = format!("{}/capture", posthog_host);
+
+                        // Define the event data
+                        let data = json!({
+                            "api_key": api_key,
+                            "event": "pg_search Deployment",
+                            "distinct_id": uuid::Uuid::new_v4().to_string(),
+                            "properties": {
+                                "commit_sha": commit_sha
+                            }
+                        });
+
+                        // Create a new HTTP client and send the event
+                        let client = reqwest::blocking::Client::new();
+                        let response = client
+                            .post(endpoint)
+                            .header("Content-Type", "application/json")
+                            .body(data.to_string())
+                            .send();
+
+                        // Check if the request was successful
+                        match response {
+                            Ok(res) if res.status().is_success() => {
+                                let _body = res.text().unwrap_or_else(|_| {
+                                    String::from("Failed to read response body")
+                                });
+                            }
+                            Ok(res) => {
```

Comment:
I don't think we should log telemetry errors to the user

Reply:
If you do choose to remove telemetry errors behind logs, I suggest you do it conditionally on a DEBUG var or something. Otherwise we're going to be debugging silent telemetry errors ourselves!

Reply:
I decided to just remove the logs, this is pretty consistent code now that it's tested and these checks are honestly a bit simplistic anyways, just checking for an env var. I don't want to complicate the code with another variable to deal with across the whole stack